lib/crossed.gi

```diff
@@ -600,10 +600,7 @@ function( R, G, act, twist )
       SetIsFiniteDimensional( RG, IsFinite( G ) );
     fi;
     
-    # What about IsCommutative ? In MagmaRings it is as below:   
-    # if HasIsCommutative( R ) and HasIsCommutative( G ) then
-    #   SetIsCommutative( RG, IsCommutative( R ) and IsCommutative( G ) );
-    # fi;
+    SetIsCommutative( RG, LeftActingDomain(RG) = Center(G) );
```

Comment:
@limakzi Actually, why `Center(G)` and not `Center(RG)`? However, that does not seem to be helpful either.